website/README.md

```diff
@@ -118,6 +118,42 @@ The code for the client is located under `src/client/`. The webpack build create
 
 Scripts related to deployment, like the Dockerfile, are under `deploy`. Note: even though the scripts are under `deploy`, you must run them from the **root of the repository**; they are sensitive to `cwd`.
 
+## Query Parameters
+
+You can pass certain parameters in the Snack URL to customize the behavior of the Snack.
+
+Here is a summary of all the parameter options that you can use:
+
+| parameter | default | description | type |
```

Comment:
Not sure how to emphasize this best, but I think it would be good to do so.

```suggestion
## Query Parameters

You can pass certain parameters in the Snack URL to customize the behavior of the Snack.

Here is a summary of all the parameter options that you can use:

> All query parameter values must be URL encoded

| parameter | default | description | type |
```